CODEOWNERS

```diff
@@ -2,3 +2,9 @@
 # described in https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/
 /sql/moz-fx-data-shared-prod/search_terms @whd @jasonthomas
 /sql/moz-fx-data-shared-prod/search_terms_derived @whd @jasonthomas
+/sql/moz-fx-data-shared-prod/fenix_derived/active_users_aggregates_v1 @mozilla/kpi_table_reviewers
```

Comment:
@bochocki preferences about the name of this group? I'll check when it's available before merging the PR.

Reply:
This sounds good to me!

Reply:
Resolved.